_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -173,6 +175,14 @@ This `input` element does not have an [explicit role][] of `textbox`, but the `a
 <label>Password<input type="password" aria-required="true"/></label>
 ```
 
+#### Passed Example 12
```

Comment:
@Jym77, correct me if I'm wrong, but per applicability this example should be inapplicable, as the aria-checked has not been explicitly set.